packages/cli/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.0.21",
   "description": "CLI devtools for the openfn toolchain.",
   "engines": {
-    "node": ">=16",
+    "node": ">=18",
```

Comment:
Just want to flag this - this branches off of #134 and so it it also requires us to bump node up 18. I think this is probably OK but need to make the risk really clear.